framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -5525,8 +5753,15 @@ VulkanReplayConsumerBase::OverrideQueuePresentKHR(PFN_vkQueuePresentKHR
     // Only attempt to find imported or shadow semaphores if we know at least one around.
     if ((!have_imported_semaphores_) && (shadow_semaphores_.empty()) && (modified_present_info.swapchainCount != 0))
     {
-        result = swapchain_->QueuePresentKHR(
-            func, capture_image_indices_, swapchain_infos_, queue_info, &modified_present_info);
+        if (queue_info == nullptr || queue_info->handle == VK_NULL_HANDLE)
```

Comment:
In the begin of `OverrideQueuePresentKHR`, it has checked `queue_info != nullptr`.  We could also check  `queue_info->handle != VK_NULL_HANDLE` there, so we don't need to check them here again.